addons/a11y/package.json

```diff
@@ -54,16 +54,16 @@
     "@storybook/core-events": "6.3.0-alpha.28",
     "@storybook/theming": "6.3.0-alpha.28",
     "axe-core": "^4.2.0",
-    "core-js": "^3.8.2",
+    "core-js": "^3.12.1",
     "global": "^4.4.0",
-    "lodash": "^4.17.20",
+    "lodash": "^4.17.21",
     "react-sizeme": "^3.0.1",
     "regenerator-runtime": "^0.13.7",
-    "ts-dedent": "^2.0.0",
+    "ts-dedent": "^2.1.1",
     "util-deprecate": "^1.0.2"
   },
   "devDependencies": {
-    "@testing-library/react": "^11.2.2",
+    "@testing-library/react": "^11.2.7",
```

Comment:
updating all these packages version to latest doesn't do storybook users any favours!

version in our package.json should be a wide as possible, so we're compatible with as many frameworks and version as possible. If we keep updating our required versions, we're making it harder for users.

We should update our lockfile, not our package.json files.

Until a new major comes out.

Can we tell renovate to only open PRs with major version bumps?